dmriprep/interfaces/images.py

```diff
@@ -1,13 +1,9 @@
 """Image tools interfaces."""
-from pathlib import Path
-
+from dmriprep.utils.images import rescale_b0, median, match_transforms, extract_b0
```

Comment:
`pathlib` is being used in the extract and rescale interfaces

```suggestion
from pathlib import Path

```